dependencies.gradle

```diff
@@ -5,7 +5,7 @@ ext {
   daggerVersion = '2.33'
   espressoVersion = '3.2.0'
   kotlinVersion = project.hasProperty('square.kotlinVersion') ?
-      project.getProperty('square.kotlinVersion') : '1.4.30'
+      project.getProperty('square.kotlinVersion') : '1.4.31'
```

Comment:
nit: commit message says this updates dagger but this commit only updates the version of kotlin.

Reply:
It's actually a 2nd commit that I squeezed in 🙈